db/mysql/ops.go

```diff
@@ -49,19 +50,24 @@ func (c *client) SendMessage(ctx context.Context, id string, message *domain.Mes
 	return nil
 }
 
-func (c *client) ListChatRoom(_ context.Context) ([]*domain.ChatRoom, error) {
+func (c *client) ListChatRoom(ctx context.Context) ([]*domain.ChatRoom, error) {
 	rows, err := c.dbc.Query("SELECT id, name FROM chatrooms")
 	if err != nil {
 		return nil, err
 	}
 
 	defer rows.Close()
 
+	return scanChatRooms(rows)
+}
+
+func scanChatRooms(rows *sql.Rows) ([]*domain.ChatRoom, error) {
```

Comment:
Move this function to the end of the file. There is a hierarchy of the function as follows: public function > private function with a receiver > private function without a receiver.

Reply:
Done